components/tooltip/README.md

```diff
@@ -69,7 +80,16 @@ The `d2l-tooltip` component is used to display additional information when users
   import '@brightspace-ui/core/components/inputs/input-text.js';
   import '@brightspace-ui/core/components/tooltip/tooltip.js';
 </script>
-
+<!-- docs: start hidden content --> 
+<script>
+  window.addEventListener('load', function () {
```

Comment:
If I'm nitting on the code samples/demos, these should all use arrow functions `() => {` but there's likely lots of existing stuff that isn't so 🤷 .

Reply:
> but there's likely lots of existing stuff that isn't so 🤷 .

Yeah so I guess your point is consistency, huh?

I mean, we could always just use arrow functions going forward at least. 
Perhaps we could add a chore-y Dragonslayer item for updating any other such code snippets to use arrow functions?

Reply:
Yeah don't worry about it for now. I might do a search & replace tomorrow.